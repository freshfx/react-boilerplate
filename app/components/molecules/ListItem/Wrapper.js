import styled from '@emotion/styled'

const Wrapper = styled.li`
  width: 100%;
  height: 3em;
  display: flex;
  align-items: center;
  position: relative;
  border-top: 1px solid #eee;

  &:first-of-type {
    border-top: none;
  }
`

export default Wrapper
