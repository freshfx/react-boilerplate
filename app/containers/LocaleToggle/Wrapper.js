import styled from '@emotion/styled'

const Wrapper = styled.div`
  padding: 2px;
`

export default Wrapper
