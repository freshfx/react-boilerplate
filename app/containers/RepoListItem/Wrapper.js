import styled from '@emotion/styled'

const Wrapper = styled.div`
  width: 100%;
  height: 100%;
  display: flex;
  align-items: space-between;
`

export default Wrapper
