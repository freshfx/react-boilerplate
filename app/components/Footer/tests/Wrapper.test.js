import 'jest-styled-components'
import React from 'react'
import {mount} from 'enzyme'

import Wrapper from '../Wrapper'

const renderComponent = (props = {}) => mount(<Wrapper {...props} />)

describe('<Wrapper />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderComponent()
    expect(renderedComponent).toMatchSnapshot()
  })
})

