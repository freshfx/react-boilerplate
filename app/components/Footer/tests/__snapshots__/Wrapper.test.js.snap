// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 3em 0;
  border-top: 1px solid #666;
}

<Wrapper>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "Wrapper-wcfdfo-0",
          "isStatic": false,
          "lastClassName": "c0",
          "rules": Array [
            "display:flex;justify-content:space-between;padding:3em 0;border-top:1px solid #666;",
          ],
        },
        "displayName": "Wrapper",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "Wrapper-wcfdfo-0",
        "target": "footer",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <footer
      className="c0"
    />
  </StyledComponent>
</Wrapper>
`;
